src/main/editDelete/preProcess.ts

```diff
@@ -19,89 +20,16 @@ const camelCase: MapCallback<SnakeCaseWord, PartialWord> = (word) => ({
 });
 
 /**
- * Injects extra attributes into a PartialWord to make it a full Word -
- * @param word the word to fill attributes for
- * @param i index of the word in the transcript
- * @returns full Word object
+ * Injects extra attributes into a PartialWord to make it a full Word
  */
 const injectAttributes: (fileName: string) => MapCallback<PartialWord, Word> =
-  (fileName: string) => (word, i) => {
-    return {
-      ...word,
-      outputStartTime: word.startTime,
-      key: i.toString(),
-      deleted: false,
-      fileName,
-    };
-  };
-
-const constructWord: (
-  word: string,
-  startTime: number,
-  duration: number,
-  outputStartTime: number,
-  key: string,
-  fileName: string
-) => Word = (word, startTime, duration, outputStartTime, key, fileName) => ({
-  word,
-  startTime: roundToMs(startTime),
-  duration: roundToMs(duration),
-  outputStartTime: roundToMs(outputStartTime),
-  deleted: false,
-  key,
-  fileName,
-});
-
-/**
- * Adds spaces between words which represent the silence between each word
- * @param result The output being constructed
- * @param word The current element of words
- * @param index The index of word in words
- * @param words The list of words being reduced
- * @returns The updated transcript with a silence after word
- */
-const addSpaces: (totalDuration: number) => MapCallback<Word, Word[]> =
```

Comment:
does the transcription object no longer use spaces to represent silences?

Reply:
No, this logic was moved to a shared file as it’s used both by live process (front end) and pre process (back end)

see processingShared.ts

However I think we may be able to simplify things and only add in spaces for (a) rendering the text and (b) video preview, which would make manipulating the transcript far easier

Reply:
Update: no longer using spaces